tests/components/smlight/snapshots/test_binary_sensor.ambr

```diff
@@ -0,0 +1,189 @@
+# serializer version: 1
```

Comment:
Please delete the snapshots and regenerate

Reply:
have regenerated